[2024-01-18T22:00:02.938+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_theta manual__2024-01-18T18:56:27.556327+00:00 [queued]>
[2024-01-18T22:00:02.966+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_theta manual__2024-01-18T18:56:27.556327+00:00 [queued]>
[2024-01-18T22:00:02.966+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:00:02.967+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T22:00:02.967+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:00:03.002+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): test_model_theta> on 2024-01-18 18:56:27.556327+00:00
[2024-01-18T22:00:03.024+0300] {standard_task_runner.py:55} INFO - Started process 5474 to run task
[2024-01-18T22:00:03.071+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'test_model_theta', 'manual__2024-01-18T18:56:27.556327+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpwivpepyw']
[2024-01-18T22:00:03.075+0300] {standard_task_runner.py:83} INFO - Job 19: Subtask test_model_theta
[2024-01-18T22:00:03.204+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.test_model_theta manual__2024-01-18T18:56:27.556327+00:00 [running]> on host xflow
[2024-01-18T22:00:03.366+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=test_model_theta
AIRFLOW_CTX_EXECUTION_DATE=2024-01-18T18:56:27.556327+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-18T18:56:27.556327+00:00
[2024-01-18T22:00:03.369+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T22:00:03.370+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/test_model.py theta']
[2024-01-18T22:00:03.399+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T22:00:11.420+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T22:00:11.488+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=test_model_theta, execution_date=20240118T185627, start_date=20240118T190002, end_date=20240118T190011
[2024-01-18T22:00:11.565+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T22:00:11.601+0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
